backend/test/mastodon.spec.ts

```diff
@@ -281,4 +282,21 @@ describe('Mastodon APIs', () => {
 		const data = await res.json<any>()
 		assert.equal(data.length, 0)
 	})
+
+	describe('Microformats', () => {
+		test('convert mentions to HTML', () => {
+			const status = 'hey @test@example.com hi'
+
+			assert.equal(
+				enrichStatus(status),
+				'<p>hey <span class="h-card"><a href="https://example.com/@test" class="u-url mention">@<span>test</span></a></span> hi</p>'
+			)
+		})
```

Comment:
Is it worth testing false positives? E.g. if something had already converted a mention to an anchor elsewhere? Do we handle that or just double enrich it?

Reply:
Links aren't safe from double enriching. We could just check for the first <p> tag to see if it's already HTML or not.